src/cascadia/WindowsTerminal/AppHost.cpp

```diff
@@ -757,7 +758,7 @@ bool AppHost::_LazyLoadDesktopManager()
 void AppHost::_HandleSummon(const winrt::Windows::Foundation::IInspectable& /*sender*/,
                             const Remoting::SummonWindowBehavior& args)
 {
-    _window->SummonWindow(args.ToggleVisibility());
+    _window->SummonWindow(args.ToggleVisibility(), args.DropdownDuration());

```

Comment:
Any reason not to just pass down the whole `args` at this point?

Reply:
I suppose not really

Reply:
Please do not make IslandWindow more specific to Terminal. I'd prefer it to be _less specific_ to Terminal IMO :smile: